tsnet/example/tshello/tshello.go

```diff
@@ -29,6 +29,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
+	defer func() {
```

Comment:
prefer two defer calls over a defer of a closure

Reply:
and then you can put the s.Close after s := and the ln.Close here

Reply:
Done, thank you @bradfitz!